File: abc003/src/a.go
package main

import "fmt"

func main() {
    var n int
    fmt.Scan(&n)
    sum := 0
    for x := 1; x <= n; x++ {
        sum += x * 10000
    }
    ans := float64(sum) / float64(n)
    fmt.Println(ans)
}

File: abc003/src/a.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;
    int sum = 0;
    for(int x = 1; x <= n; x++) {
        sum += x * 10000;
    }
    double ans = static_cast<double>(sum) / n;
    cout << ans << endl;
    return 0;
}